### ISSUE TITLE
[CLEANUP] Use setup.cfg instead of separate config files

### ISSUE THREAD

**Issue by ObaraEmmanuel** [pesos/browser-history#163]
Currently, the root of the project is bloated with configuration files such as

- [.flake8](.flake8)
- [.coveragerc](.coveragerc)
- [.pylintrc](.pylintrc)
- [pytest.ini](pytest.ini)

Most of these configurations can be provided through a unified `setup.cfg` but there is a catch, you need to do a little research from the vendors of the specific tools that need these configurations. For instance, 
`pytest.ini`
```ini
[pytest]
markers =
    browser_name: browser name used in default browser tests
```
can be replaced with the following in `setup.cfg`
```ini
[tool:pytest]
markers =
    browser_name: browser name used in default browser tests
```

And you can add more configurations for different tools in the same `setup.cfg` file. They may require different header like `[tool:pytest]` for `pytest.ini` .The task is in finding  out which header format to use for which configuration and move the configuration to the `setup.cfg`. 

> Note:
> Some tools may not support use of the `setup.cfg` in which case you can leave the configuration as is

Happy coding.

**Comment by xames3** [comment]
Hello @Samyak2 and @ObaraEmmanuel, I would like to work on this issue.

**Comment by Samyak2** [comment]
@xames3 sure, go ahead!

**Comment by xames3** [comment]
Do we still need to keep this issue open @Samyak2?

**Comment by Samyak2** [comment]
Fixed via #167 

Thanks for noticing
